### PR TITLE
landing page mobile

### PR DESCRIPTION
**Pull request by mruwnik**

### CODE REVIEW

File: app/components/ContentBox/box.css [hunk 1]
```diff
@@ -52,3 +52,15 @@
   padding: 0;
   padding-right: var(--spacing-8);
 }
+
+@media (max-width: 600px) {
+  .main-container-box-table {
+    flex-direction: column;
+    height: 550px;
+    margin-bottom: var(--spacing-40);
```
Comment (melissasamworth) [inline app/components/ContentBox/box.css:60]: Looks good but remember that --spacing-40 turns into I think 24px on mobile, so hope you were trying to make this appear 24px!
image

File: app/components/Footer/footer.css [hunk 2]
```diff
@@ -10,3 +10,11 @@
   align-items: baseline;
   margin: var(--spacing-104) 13.33vw var(--spacing-80) 13.33vw;
 }
+
+@media (max-width: 600px) {
+  .footer-contents {
+    flex-direction: column;
+    gap: var(--spacing-32);
```
Comment (melissasamworth) [inline app/components/Footer/footer.css:17]: I think using spacing-80 var, making is 40px for mobile. Sorry for my not super organized system, will try to make it better soon.

File: app/components/Grid/grid.css [hunk 3]
```diff
@@ -1,9 +1,16 @@
...
+  flex-flow: wrap;
+}
+
 .grid-item {
   cursor: pointer;
   text-align: left;
   padding: var(--spacing-32) var(--spacing-32);
-  min-width: var(--spacing-288);
-  min-height: var(--spacing-288);
+  width: var(--spacing-288);
```
Comment (melissasamworth) [inline app/components/Grid/grid.css:12]: I think these should be based on viewport width. I generally won't use variables to define content that is not viewport width (e.g. popovers).

File: app/newRoot.css [hunk 4]
```diff
@@ -304,6 +304,9 @@ ul {
   flex-grow: 2;
 }
 
+.centered {
```
Comment (melissasamworth) [inline app/newRoot.css:307]: thanks for using more one-off class names

File: app/newRoot.css [hunk 5]
```diff
@@ -348,3 +351,53 @@ ul {
...
+  display: inherit;
+}
+
+.mobile-only {
+  display: none;
+}
+
+@media (max-width: 600px) {
+  .page-body {
+    padding: 0px 6.333vw;
```
Comment (melissasamworth) [inline app/newRoot.css:365]: cool yeah, and let's make sure we use `.page-body` everywhere that uses standard margins, including the footer actually

**Review by melissasamworth — CHANGES_REQUESTED** [review]

File: app/newRoot.css (hunk 5 above)
Comment (mruwnik) [reply app/newRoot.css:365]: I actually wanted to delete this before pushing :/ I only had it for testing...

File: app/components/Grid/grid.css (hunk 3 above)
Comment (mruwnik) [reply app/components/Grid/grid.css:12]: won't that get hairy with very small screen? Also, do you have specific values? :D

File: app/components/ContentBox/box.css [hunk 6]
```diff
@@ -57,3 +57,15 @@
   padding: 0;
   padding-right: var(--spacing-8);
 }
+
+@media (max-width: 600px) {
```
Comment (melissasamworth) [inline app/components/ContentBox/box.css:61]: We're using 640

File: app/newRoot.css [hunk 7]
```diff
@@ -348,3 +351,49 @@ ul {
...
+
+.desktop-only {
+  display: inherit;
+}
+
+.mobile-only {
+  display: none;
+}
+
+@media (max-width: 600px) {
```
Comment (melissasamworth) [inline app/newRoot.css:363]: 640px

**Review by melissasamworth — APPROVED** [review]

**Pull request merged**